strapi/config/admin.ts

```diff
@@ -0,0 +1,8 @@
+export default ({ env }) => ({
+  auth: {
+    secret: env('ADMIN_JWT_SECRET'),
+  },
+  apiToken: {
+    salt: env('API_TOKEN_SALT'),
```

Comment:
This seems standard so answering me with a pointer to doc is fine, but what is this salt about ? (I understand cryptographic purpose of salt, but why is it stored in env? Isn't it better to serialize it with the secret/hashed secret as we would do with PBKDF2?)

Reply:
https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/api-tokens.html
Not sure about the specific implementation, but this is what the strapi docs state. In our case we can easily generate keys in the `.env` for dev purposes, and we can seed the secrets into the production Docker environment, so we'll probably use the secrets provider of the orchestration platform we'll use (Docker swarm secrets, HashiCorp Vault, `<insert your k8s distro here>`, ...)